cmd/sealer/cmd/cluster/run.go

```diff
@@ -115,6 +117,30 @@ func NewRunCmd() *cobra.Command {
 					app, extension, nil, imageEngine, runFlags.Mode)
 			}
 
+			if exist := osi.IsFileExist(common.GetDefaultClusterfile()); exist {
```

Comment:
not use file existing  to do this pre-check, try get  it from `clusterfile.NewClusterFile(nil)`,
 if not found, no cluster.
 if have ,determine whether the image name is the same one.